tests/components/zwave/test_init.py

```diff
@@ -1027,6 +1040,15 @@ def test_set_config_parameter(self):
 
         assert value_list.data == 'item3'
 
+        self.hass.services.call('zwave', 'set_config_parameter', {
+            const.ATTR_NODE_ID: 14,
+            const.ATTR_CONFIG_PARAMETER: 15,
+            const.ATTR_CONFIG_VALUE: '3',
```

Comment:
We want to pass this in as an integer, to test that it is casted correctly, right?

Reply:
Yup. My mistake :)